CHANGES

```diff
@@ -1,4 +1,45 @@
 
+Changes with Unit 1.32.0                                         27 Feb 2024
+
+    *) Feature: conditional access logging.
+
+    *) Feature: NJS variables access.
+
+    *) Feature: $request_id variable contains a string that is formed using
+       random data and can be used as a unique request identifier.
+
+    *) Feature: options to set control socket permissions.
+
+    *) Feature: Ruby response field arrays.
+
+    *) Bugfix: router could crash while sending large files. Thanks to
+       rustedsword.
+
+    *) Bugfix: trying to serve static files over a certain size could lead
+       to error.
```

Comment:
Not so much files over a certain size, but serving from a network filesystem for example. Anything where a read(2) could return less bytes than asked for...

Reply:
Ok. What do you think about this alternative:

`Bugfix: serving static files from a network filesystem could lead to error.`